Vizzuality/half-earth-v3
# Fix/shapefile text

## Update text. Remove landscape mode
### Description

- Update shapefile text and error
- Remove landscape mode
- Remove unused map-iframe
### Testing instructions
- Check AOI shapefile action text
- Go to feature globe and zoom. You shouldn't see landscape mode
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-276?atlOrigin=eyJpIjoiNjI3ZDFlNGIxMmM5NDg4ZTlmNGU5ODZlNzU1YzgwYjgiLCJwIjoiaiJ9
https://vizzuality.atlassian.net/browse/HE-308?atlOrigin=eyJpIjoiNDVmNDYzOGUwMDhlNDI0NDlkYjBkNDRhNzFhNWExY2UiLCJwIjoiaiJ9
